README.md

```diff
@@ -124,7 +139,64 @@ mvn test -Dtest=SanityTests -X
         
   </details>
 
-<br/><br/>
+</br>
+
+> If you weren't successful at setting up you local env, then use the [Gitpod approach](#gitpod-setup)👇
+
+
+### Gitpod setup
+
+Open forked repo in Gitpod.io
+
+1. Sign up for a free [Github account](https://github.com/)
+2. Fork this repository
+    * Make sure you are logged into Github
+    * Click the fork in the upper right of Github
+    * Select your username as the location to fork the repo
+3. In the browser address bar, prepend the Github url (`https://github.com/<username>/automation-best-practices-java`) with `https://gitpod.io/#`
+    * The resulting url should look as follows:
+   
+      > gitpod.io/#https://github.com/{USERNAME}/automation-best-practices-java
+   
+4. Once the Gitpod.io URL is loaded, you will need to sign in with the Github account you created earlier 
+5. Once the development environment is loaded and you see 'Ready to test!', run the following commands to set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`:
+
+```bash
+eval $(gp env -e SAUCE_USERNAME=<sauce_username>)
+eval $(gp env -e SAUCE_ACCESS_KEY=sauce_access_key>)
```

Comment:
Let's also mention how they can get the user/key https://docs.saucelabs.com/basics/environment-variables/ (I might have referenced this elsewhere in the README because it obviously pertains to local env setup as well)

Reply:
@nadvolod I added a section about where to get your Sauce username and access key as well as your Screener API key. This is only in the Gitpod section, so I can move it somewhere else if you feel that is appropriate.